File: LeapYear/LeapYearKata/LeapYearKata/LeapYear.cs

namespace LeapYearKata
{
    public class LeapYear
    {
        public bool IsLeapYear(int year)
        {
            if (year%4 == 0 && !(year % 100 == 0))
            {
                return true;
            }
            return false;
        }
    }
}

File: Javascript/LeapYear/leapyear.js
exports.isLeapYear = function (year) {

    return    (year.isDivisibleBy(4) && !year.isDivisibleBy(100))
            || year.isDivisibleBy(400);
}

Number.prototype.isDivisibleBy = function (denominator) {
    return this % denominator == 0;
}